spec/fixtures/controllers/supplementals/index.json

```diff
@@ -91,371 +91,454 @@
-        },
-        "DeveloperName": "AG"
+        }
       },
-      "LW_Type_of_Proof": null,
-      "City": null,
       "State": null,
       "Street": null,
+      "Total_Household_Rent": 0.0,
+      "Type_of_proof": null,
       "Zip_Code": null,
-      "Post_Lottery_Validation": null,
-      "Total_Household_Rent": 0.0
-    },
-    {
       "attributes": {
         "type": "Application_Preference",
-        "url": "/services/data/v43.0/sobjects/Application_Preference/a0w0P00000Oqew4QAB"
+        "url": "/services/data/v43.0/sobjects/Application_Preference/a0w0P00000Oqew2QAB"
+      }
+    },
+    {
+      "Application_Member": {
+        "Date_of_Birth": "1989-01-05",
+        "First_Name": "Automated Test Lease Up Application",
+        "Id": "a0n0P00000D3qIZQAZ",
+        "Last_Name": "(do not modify)",
+        "attributes": {
+          "type": "Application_Member",
+          "url": "/services/data/v43.0/sobjects/Application_Member/a0n0P00000D3qIZQAZ"
+        }
       },
+      "Certificate_Number": null,
+      "City": null,
       "Id": "a0w0P00000Oqew4QAB",
+      "Individual_preference": null,
+      "LW_Type_of_Proof": null,
+      "Listing_Preference_ID": "a0l0P00001QRQKlQAP",
+      "Lottery_Status": "None",
       "Name": "AP-0000919889",
-      "Preference_Name": "Artist Fund (Test Custom Pref)",
-      "Person_who_claimed_Name": null,
-      "Type_of_proof": null,
       "Opt_Out": false,
-      "Lottery_Status": "Unconfirmed",
+      "Person_who_claimed_Name": "Automated Test Lease Up Application (do not modify)",
+      "Post_Lottery_Validation": null,
       "Preference_Lottery_Rank": null,
-      "Listing_Preference_ID": "a0l0P00001QRQKlQAP",
-      "Receives_Preference": false,
-      "Application_Member": null,
-      "Individual_preference": null,
-      "Certificate_Number": null,
+      "Preference_Name": "Artist Fund (Test Custom Pref)",
       "Preference_Order": 7.0,
+      "Receives_Preference": true,
       "RecordType": {
+        "DeveloperName": "Custom",
         "attributes": {
           "type": "RecordType",
           "url": "/services/data/v43.0/sobjects/RecordType/0120P000000R7BIQA0"
-        },
-        "DeveloperName": "Custom"
+        }
       },
-      "LW_Type_of_Proof": null,
-      "City": null,
       "State": null,
       "Street": null,
+      "Total_Household_Rent": 0.0,
+      "Type_of_proof": null,
       "Zip_Code": null,
-      "Post_Lottery_Validation": null,
-      "Total_Household_Rent": 0.0
+      "attributes": {
+        "type": "Application_Preference",
+        "url": "/services/data/v43.0/sobjects/Application_Preference/a0w0P00000Oqew4QAB"
+      }
     }
   ],
   "proof_files": [
     {
-      "Id": "00P0P00000t0P3vUAE",
-      "Document_Type": "Copy of Lease",
-      "Related_Application": "a0o0P00000GZazOQAT",
-      "Related_Application_Preference": "a0w0P00000OUJc2QAH"
+      "document_type": "Copy of Lease",
```

Comment:
change from SOQL to domain format. Other changes are due to changes made on the test application since we last captured the cassette